dev/breeze/src/airflow_breeze/commands/testing_commands.py

```diff
@@ -285,6 +286,13 @@ def run_tests_in_parallel(
                     if test_type.startswith(heavy_test_type):
                         test_types_list.remove(test_type)
                         tests_to_run_sequentially.append(test_type)
+            if full_tests_needed:
```

Comment:
Should this be `if not full_tests_needed`? We want to still run them when `full_tests_needed` is true, right?

Reply:
Obviously :facepalm: . That would actually explain why they were run last time :)